sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestObjectSerializer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Serialization;
+
+namespace Azure.DigitalTwins.Core.Tests
+{
+    /// <summary>
+    /// Custom serializer to pass to the DigitalTwins client.
+    /// </summary>
+    public class TestObjectSerializer : ObjectSerializer
+    {
+        private static JsonObjectSerializer _serializer = new JsonObjectSerializer();
```

Comment:
Can you add a check that the expected methods in here are actually called? Just to make sure that when a user sets the custom serializer, that it actually gets used?

Reply:
Very good idea! Let me figure out a way to do that. Or i'll add some custom implementation to ensure that was called.